app/views/layouts/application.html.erb

```diff
@@ -5,12 +5,15 @@
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
+  <% if current_user %>
```

Comment:
Consider ```<% if user_signed_in? %>``` too, but only if you find it more readable. Like the `current_user` helper, `user_signed_in?` is provided by devise.